src/coefplot.jl

```diff
@@ -11,10 +11,17 @@ function coefplot!(f::Figure, x::Union{MixedModel, MixedModelBootstrap}; conf_le
 	ax = Axis(f[1, 1])
     scatter!(ax, xvals, y)
     errorbars!(ax, xvals, y,  xvals .- ci.lower, ci.upper .- xvals, direction=:x)
-    ax.xlabel = "Estimate and $(conf_level * 100)% confidence interval"
+    ax.xlabel = "Estimate and $(round(Int, conf_level * 100))% confidence interval"
```

Comment:
Does it always need to be rounded to an integer percentage?  I can't say I want a 97.5% confidence interval?

Reply:
I was actually wondering that myself. I mostly wanted to avoid `95.0%` percentage showing up. Maybe it's best to just not do any rounding for now?

Reply:
How about using Printf.@sprintf?
```julia
julia> using Printf

julia> fmt = @sprintf "%g" 95.0
"95"

julia> fmt = @sprintf "%g" 97.5
"97.5"
```

Reply:
good idea and done :smile: